### DISCUSSION TITLE
CUDA problems

### DISCUSSION THREAD

**Discussion by Lautaro22** [alicevision/Meshroom#1854] [Q&A]
Hi! In the "deep Map" step the program show me this error message : 
[11:45:54.850454][error] cudaGetDeviceCount failed: CUDA driver version is insufficient for CUDA runtime version
and 
[11:45:54.850454][error] This program needs a CUDA-Enabled GPU (with at least compute capability 2.0).

i have installed this CUDA version:
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Mon_Oct_24_19:40:05_Pacific_Daylight_Time_2022
Cuda compilation tools, release 12.0, V12.0.76
Build cuda_12.0.r12.0/compiler.31968024_0

And I have this system
Procesador	Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz   1.19 GHz
RAM instalada	8,00 GB (7,75 GB utilizable)
Tipo de sistema	Sistema operativo de 64 bits, procesador x64

Can I solve the problem?
Thank for your responses

**Comment by natowi** [comment]
What GPU model do you have? Did you install the latest nvidia drivers?

**Comment by Lautaro22** [comment]
Hi! thanks for your response!
I have this GPU:
![imagen](https://user-images.githubusercontent.com/16881950/210334805-698061e0-b17b-4b45-bec5-2560f9a58764.png)
I install the last version but I not sure is the installation was done well

**Reply by natowi** [reply]
For the DepthMap node a NVIDIA GPU is required. You can use https://github.com/alicevision/Meshroom/wiki/Draft-Meshing
